packages/core/admin/ee/server/services/passport.js

```diff
@@ -5,14 +5,25 @@ const { features } = require('@strapi/strapi/lib/utils/ee');
 
 const createLocalStrategy = require('../../../server/services/passport/local-strategy');
 const sso = require('./passport/sso');
+const { isSsoLocked } = require('../utils/sso-lock');
 
-const getPassportStrategies = () => {
-  const localStrategy = createLocalStrategy(strapi);
+const localStrategyMiddleware = async ([error, user, message], done) => {
+  if (await isSsoLocked(user)) {
+    return done(error, null, {
+      message: 'Login not allowed, please contact your administrator',
```

Comment:
Can we make this customizable or use the locales so it can be overridden?

Reply:
@simotae14 I've updated this so that when it happens the front-end receives the following:

```
{
    "data": null,
    "error": {
        "status": 401,
        "name": "UnauthorizedError",
        "message": "Login not allowed, please contact your administrator",
        "details": {
            "code": "DISALLOWED"
        }
    }
}
```

Can you catch that error.details.code on the [front-end](https://github.com/strapi/strapi/pull/16765) and display a localized message?

Reply:
sure

Reply:
Are you sure you have the latest version? This is what I get sending the login from postman:

<img width="705" alt="image" src="https://github.com/strapi/strapi/assets/999278/fd8b4c26-46e9-4eda-8530-1a22046075b5">